source/install/cloud-changelog.md

```diff
@@ -4,12 +4,70 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-06-15](#release-2022-06-15)
 - [Release 2022-05-26](#release-2022-05-26)
 - [Release 2022-05-12](#release-2022-05-12)
 - [Release 2022-04-28](#release-2022-04-28)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
-- [Release 2022-03-16](#release-2022-03-16)
+
+## Release 2022-06-15
+
+### Highlights
+
+#### Free Forever Mattermost Cloud Plan
+ - Mattermost Cloud now supports a free forever plan, Cloud Starter, for unlimited users for unlimited time.
+ - Access unlimited channels, playbooks, and boards across the Mattermost platform, with unlimited voice calls and screen sharing in Direct Messages.
+ - The free plan includes a few workspace limits:
+    - Maximum 1 team. Any additional teams created during a trial or on a paid plan will be archived.
+    - 10GB file storage across the platform, with 100MB upload limit.
+    - Unlimited installed apps or plugins, with maximum of 5 enabled at one time.
+    - Access to 10,000 most recent messages.
```

Comment:
I'd mention that history will be preserved

Reply:
I believe history is preserved across all 4 limits above as well, though that archived history can only be accessed if a free user upgrades to a paid subscription plan.